payout/client.go

```diff
@@ -26,13 +27,17 @@ func (c *Client) CreateWithContext(ctx context.Context, data *CreateParams) (*xe
 	}
 
 	response := &xendit.Payout{}
+	header := http.Header{}
+	if data.IdempotencyKey != "" {
+		header.Add("X-IDEMPOTENCY-KEY", data.IdempotencyKey)
```

Comment:
Could we put a lower case header `x-idempotency-key` here? I know on the docs its capitalized but I think this will make it consistent with our other headers. Thanks Ricky!

Reply:
sure thing! 
fixed here : 8c97da68613789246dc6249af498804a08bfb7ea

📝 im leaving a note that the other clients (card and disbursement) are still having uppercase idempotency key header
https://github.com/xendit/xendit-go/blob/8a3d6f222ccdc8ede49b786e7c41d8ed111a9543/disbursement/client.go#L33
https://github.com/xendit/xendit-go/blob/8a3d6f222ccdc8ede49b786e7c41d8ed111a9543/disbursement/client.go#L161
https://github.com/xendit/xendit-go/blob/8a3d6f222ccdc8ede49b786e7c41d8ed111a9543/card/client.go#L125

Reply:
Ah I see, thanks for the shout @rjjatson. Will make a note of this to fix in the future